ansible/roles/ocp4-workload-integreatly/files/threescale/delete-tenant.yml

```diff
@@ -0,0 +1,63 @@
+---
```

Comment:
Do we need this delete logic?

Reply:
Not sure @aidenkeating and @pmccarthy ?

Reply:
I wouldn't think so as any tenant data will be stored in-cluster meaning that once the cluster gets removed, so will all generated tenants. Should be fine to leave out

Reply:
:+1: